### DISCUSSION TITLE
Lets do something toghether?

### DISCUSSION THREAD

**Discussion by MesumRaza** [ThioJoe/SRT-To-SSML#5] [General]
I am working on the first part where I take a Video and Get it Word by Word Transcribed via DeepGram and get an SRT, You can add second leg and translate that video in english by your project.

**Comment by ThioJoe** [comment]
I have another project that might interest you: https://github.com/ThioJoe/Auto-Synced-Translated-Dubs

It translates the subtitles into several languages, creates dubbed audio with AI generated voice from Azure, and syncs the dubbed audio up with the subtitles.

**Reply by MesumRaza** [reply]
Yes I have seen that. In the same context we can build some thing together something like.

Transcribe your Video, Translate your Video and also Sync Wav2Lip.

So it will be very Natural If I make a Tutorial in Urdu and I can translate that into English Tutorial

We can also do voice cloning with Coqui TTS Stuff
